rasa_core/server.py

```diff
@@ -432,6 +451,13 @@ def load_model():
         logger.debug("Finished loading new agent.")
         return jsonify({'success': 1})
 
+    @app.route("/model/time-of-last-train", methods=['GET'])
+    @requires_auth(auth_token)
```

Comment:
This should rather use a property dumped to the metadata when persisted. The model might not habe been trained on this rasa core instance

Reply:
well the property is on the agent, so should i just include this property when persisting the agent?

Reply:
The issue is that if you load a persisted model, the time-of-last-train will always be None, even if you load a different model.

My idea was, to persist a uuid, timestamp, ... as part of the metadata https://github.com/RasaHQ/rasa_core/blob/master/rasa_core/policies/ensemble.py#L139 so we can use it when loading the model.

Reply:
that make sense!